blog/.vuepress/theme/layouts/Post.vue

```diff
@@ -13,7 +13,18 @@
 
           <div class="post-full-byline">
             <section class="post-full-byline-content">
+              <ul class="author-list" v-if="current.author && gravatar">
```

Comment:
I would suggest to have a `author` as a store value and extract the logic `current.author && gravatar`.